src/main/java/spoon/support/compiler/jdt/JDTCommentBuilder.java

```diff
@@ -373,6 +375,14 @@ public void visitCtAnonymousExecutable(CtAnonymousExecutable e) {
 			public <T> void visitCtParameter(CtParameter<T> e) {
 				e.addComment(comment);
 			}
+
+			@Override
+			public void visitCtCatch(CtCatch e) {
+				if (comment.getPosition().getLine() <= e.getPosition().getLine()) {
```

Comment:
What I understand is that `sourceStart` starts at the same position than the first comment, which means the equality is respected and the comment is included (it was the previous implementation). But then the sourceStart of the second comment is greater than this sourcePosition and so it's not considered. 
Now with getLine either it considers all are on the same line, either it computes well the different lines using the line separator position, which is the case I think.